packages/core/src/plugin/index.ts

```diff
@@ -1,15 +1,14 @@
-import { isArray, isObject } from '../utils/index.js';
 import { convertPlugin } from './js/index.js';
 import { rustPluginResolver } from './rust/index.js';
-
 import type { JsPlugin } from './type.js';
 import { ResolvedUserConfig, type UserConfig } from '../config/index.js';
 import merge from '../utils/merge.js';
 
 export * from './js/index.js';
 export * from './rust/index.js';
 
-export async function resolveFarmPlugins(config: UserConfig) {
+// combing resolveFarmPlugin and resolveAsyncPlugin
+export async function resolveFarmPluginsAndAsyncPlugins(config: UserConfig) {
```

Comment:
`packages/core/src/config/index.ts:13` references it but does not sync the changes

Reply:
Could you elaborate the what you mean by references it?

My understanding, you call the function it has synchronized issue.

Reply:
You changed its name but did not change it synchronously in `packages/core/src/config/index.ts` where it is used.

- https://github.com/farm-fe/farm/blob/main/packages/core/src/config/index.ts#L7-L14
- https://github.com/farm-fe/farm/blob/main/packages/core/src/config/index.ts#L168-L172